library/TileTypes.cpp

```diff
@@ -340,6 +340,11 @@ namespace DFHack
                 // Direction is medium value match
                 if (cur_direction == tileDirection(tt))
                     value |= 4;
+                
```

Comment:
please remove trailing whitespace

Reply:
Everytime. I will go ask someone if visual studio can automagic that for me, I clearly can't be trusted with remembering this.